app/jinja_filters.py

```diff
@@ -364,11 +364,11 @@ def __init__(
 
 
 @blueprint.app_template_filter()  # type: ignore
-def map_select_config(form: FormType, answer: AnswerType) -> list[SelectConfig]:
-    options = form["fields"][answer["id"]]
+def map_select_config(form: FormType, answers: AnswerType) -> list[SelectConfig]:
+    options = form["fields"][answers["id"]]
```

Comment:
How come this changed, is it still singular?.

Reply:
I changed it because it could now accept multiple so I just thought it made more sense

Reply:
The answer type and code doesn't reflect that. You can't do key access on an array. It is still one answer?

Reply:
Ya I think that make most sense, the options var is already plural.